core/spec/models/spree/order_spec.rb

```diff
@@ -40,6 +40,28 @@ def compute(_computable)
     end
   end
 
+  describe "#cancel!", pending: true do
+    context "with captured store credit" do
+      let!(:store_credit_payment_method) { create(:store_credit_payment_method) }
+      let(:order_total) { 500.00 }
+      let(:store_credit) { create(:store_credit, amount: order_total) }
+      let(:order) { create(:order_with_totals, user: store_credit.user, line_items_price: order_total) }
```

Comment:
This factory is crap (sorry). The `order_with_totals` factory actually makes an order with no totals :disappointed_relieved: :rage4: 

As a result this spec wasn't actually failing after you removed the `.update!` :cry:.

Changing this to `create(:order_with_line_items` should fix the problem.